toolbox/arcgee/data.py

```diff
@@ -1054,6 +1054,27 @@ def init_and_set_tags(project=None, workload_tag=None):
     return
 
 
+# Get ROI from the object extent.
+# Use string-based type hints to avoid type checking errors in ArcGIS Pro 3.2.
+def get_roi_from_object(obj: "ee.Image | ee.FeatureCollection") -> "ee.Geometry.BBox":
+    """Get the ROI from the object extent.
+
+    Args:
+        obj (ee.Image or ee.FeatureCollection): Earth Engine object
```

Comment:
* Unnecessary type in the arg description as there is a type annotation.
* object --> `ComputedObject`